app/apollo/resolvers/channel.js

```diff
@@ -598,6 +620,18 @@ const channelResolvers = {
           }
         } ) );
 
+        // Create output for graphQL plugins
+        const subscriptionFind = await models.Subscription.find({org_id});
+        const subscriptionObjs = _.map(subscriptionFind, (subscription)=>{
+          return {
+            name: subscription.name,
+            uuid: subscription.uuid,
+          };
+        });
```

Comment:
This finds all subscriptions in the org, not just those affected by this particular action. Instead of needing a new query, the subscriptions should just be built by the processing of the earlier Promise.all()'.